integreat_cms/cms/forms/pages/page_translation_form.py

```diff
@@ -84,8 +85,10 @@ def save(self, commit=True):
             self.instance.version += 1
             self.instance.pk = None
 
-        # Save CustomModelForm
-        return super().save(commit=commit)
+        # Save CustomModelForm and flush Cache
+        result = super().save(commit=commit)
+        invalidate_model(PageTranslation)
+        return result
```

Comment:
I'm not sure, I think this is also implicitly done because the page translation is saved together with a page and saving a page triggers cache invalidation through the move method? Also, I'm not sure whether we really should invalidate the whole model here - maybe it's enough to invalidate all parents and children in case the slug changed (because urls might have changed?)...

Reply:
Okay, lets drop this.